src/main/scala/epic/dense/LowRankQuadraticTransform.scala

```diff
@@ -0,0 +1,142 @@
+package epic.dense
+
+import breeze.linalg._
+import breeze.linalg.operators.OpMulMatrix
+import epic.features.SegmentedIndex
+import epic.framework.Feature
+import breeze.util.Index
+
+import scala.runtime.ScalaRunTime
+import scala.util.Random
+
+case class LowRankQuadraticTransform[FV](numOutputs: Int, numRanks: Int, numLeftInputs: Int, numRightInputs: Int, innerTransform: Transform[FV, DenseVector[Double]]) extends OutputTransform[FV, DenseVector[Double]] {
```

Comment:
quick doc comment. is this this the MIT stuff? I didn't have much luck with that...